### PR TITLE
Update example app and add guide for its use

### PR DESCRIPTION
**Pull request by robszumski**
Supercedes #133

### CODE REVIEW

File: docs/guide-first.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+First, SSH to your EC2 machine:
+
+```sh
+$ ssh ec2-user@<ip address>
+```
+
+Install the Nitro Enclave packages:
+
+```
+$ sudo amazon-linux-extras install aws-nitro-enclaves-cli -y
```
Comment (eyakubovich) [inline docs/guide-first.md:29]: `sudo yum install -y amazon-linux-extras aws-nitro-enclaves-cli` - right?

File: docs/guide-first.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+```yaml
+version: v1
+name: "example"
+target: "enclave:latest"
+sources:
+  app: "app:latest"
+defaults:
+  memory_mb: 1000
+egress:
+  - host
```
Comment (eyakubovich) [inline docs/guide-first.md:89]: You don't need the whole `egress` key. The egress policy is only needed if the code inside the enclave issues a `connect`.

File: docs/guide-first.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+$ git clone https://github.com/edgebitio/enclaver
+```
+
+Download the latest Enclaver binary:
+
+```sh
+$ curl -sL $(curl -s https://api.github.com/repos/edgebitio/enclaver/releases/latest | jq -r '.assets[] | select(.name|match("^enclaver-linux-x86_64(.*)tar.gz$")) | .browser_download_url') --output enclaver.tar.gz
+$ tar -xvf enclaver.tar.gz
+$ chmod +x enclaver-linux-x86_64-v0.2.0/enclaver
+$ sudo cp enclaver-linux-x86_64-v0.2.0/enclaver /usr/bin/enclaver
```
Comment (eyakubovich) [inline docs/guide-first.md:53]: Let's put it into `/usr/local/bin`

Comment (eyakubovich) [reply docs/guide-first.md:53]: I guess it won't work b/c with `sudo`, `/usr/local/bin` is not in the `PATH`

File: docs/guide-first.md (hunk 1 above)
Comment (robszumski) [reply docs/guide-first.md:29]: yeah it worked either way

**Review by eyakubovich — APPROVED** [review]

**Pull request merged**